packages/hummingbird/src/templates/components/CampaignConfirmed/DefaultText.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Title from './Title';
+import Text from './Text';
+
+export default () => (
+  <React.Fragment>
+    <FormattedMessage
+      id="hummingbird.CampaignConfirmed.main.title"
+      description="Title of campaign submission success."
+      defaultMessage="Hurray!"
+      tagName={Title}
+    />
+    <FormattedMessage
+      id="hummingbird.CampaignConfirmed.main.text"
+      description="Text of campaign submission success."
+      defaultMessage="Thanks to you, we are one step closer towards our goals."
+      tagName={Text}
+    />
+  </React.Fragment>
+);
```

Comment:
I wonder if we need a DefaultText, as all campaign types need to have independent customizable copy.

Reply:
Added different texts/translations for the different campaign types.

Maybe you can add some good Dutch translations of the strings before merging? ;)